plonk-core/src/proof_system/linearisation_poly.rs

```diff
@@ -25,26 +26,18 @@ use ark_serialize::{
     CanonicalDeserialize, CanonicalSerialize, Read, SerializationError, Write,
 };
 
-/// Polynomial Evaluations
-///
-/// This `struct` keeps track of polynomial evaluations at points `z` and/or `z
-/// * w` where `w` is a root of unit.
-pub struct Evaluations<F>
-where
-    F: Field,
-{
-    /// Proof-relevant Evaluations
-    pub proof: ProofEvaluations<F>,
-
-    /// Evaluation of the linearisation sigma polynomial at `z`.
-    pub quot_eval: F,
-}
+use super::{
```

Comment:
The split into `use crate` an `use super` is a bit strange? `CustomValues` should be just after `WitnessValues`?

Reply:
You're right, its a bit odd. I added these automatically with the IDE and didn't notice the inconsistency. 
I'll fix it right away! :)